docs/reference/commandline/ps.md

```diff
@@ -59,7 +99,66 @@ The currently supported filters are:
     48ee228c9464        fedora:20         bash                   2 weeks ago         Exited (0) 2 weeks ago                              tender_torvalds
 
-This shows all the containers that have exited with status of '0'
+This shows all the containers that have exited successfully, i.e., with exited code of '0'.
+
+#### Status
+
+The `status` filter allows you to list containers filtered on the status of the container. You can filter using `created`, `restarting`, `running`, `paused` and `exited`.
+
+    $ docker ps --filter status=running
+    CONTAINER ID        IMAGE                  COMMAND             CREATED             STATUS              PORTS               NAMES
+    715ebfcee040        busybox                "top"               16 minutes ago      Up 16 minutes                           i_am_nostalgic
+    d5c976d3c462        busybox                "top"               23 minutes ago      Up 23 minutes                           top
+    9b6247364a03        busybox                "top"               24 minutes ago      Up 24 minutes                           nostalgic_stallman
+
+This will list all containers that are running.
+
+    $ docker ps --filter status=paused
+    CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS                      PORTS               NAMES
+    673394ef1d4c        busybox             "top"               About an hour ago   Up About an hour (Paused)                       nostalgic_shockley
+
+This will list all containers that are paused.
+
+#### Ancestor
+
+The `ancestor` filter allows to filter containers based on the image or a descendant of it.
+
+    $ docker ps --filter ancestor=ubuntu
+    CONTAINER ID        IMAGE               COMMAND             CREATED              STATUS              PORTS               NAMES
+    919e1179bdb8        ubuntu-c1           "top"               About a minute ago   Up About a minute                       admiring_lovelace
+    5d1e4a540723        ubuntu-c2           "top"               About a minute ago   Up About a minute                       admiring_sammet
+    82a598284012        ubuntu              "top"               3 minutes ago        Up 3 minutes                            sleepy_bose
+    bab2a34ba363        ubuntu              "top"               3 minutes ago        Up 3 minutes                            focused_yonath
+
+
+This will list all containers that are based on the `ubuntu` (`lastest` tag is implicit) or a descendant image of `ubuntu`.
+
+    $ docker ps --filter ancestor=ubuntu-c1
+    CONTAINER ID        IMAGE               COMMAND             CREATED              STATUS              PORTS               NAMES
+    919e1179bdb8        ubuntu-c1           "top"               About a minute ago   Up About a minute                       admiring_lovelace
+
+This will list only the containers that are based on the `ubuntu-c1` image (which, in this case is a child of `ubuntu`).
```

Comment:
Move before example. And

Match containers based on the the `ubuntu-c1` image which, in this case, is a child of `ubuntu`: